schematic_api/api/routes.py

```diff
@@ -234,7 +239,15 @@ def parse_bool(str_bool):
         raise ValueError(
             "String boolean does not appear to be true or false. Please verify input."
         )
-        
+
+def return_as_json(manifest_local_file_path):
+    manifest_csv = pd.read_csv(manifest_local_file_path)
+    try:
+        manifest_json = json.loads(manifest_csv.to_json(orient="records"))
```

Comment:
Is there a reason you are doing this instead of doing?

```
manifest_csv.to_dict(orient="records")
```

Reply:
I just tried what you suggested and it worked too. Thanks Tom!